config/cdc/postgres-entrypoint.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -e
+
+WAL2JSON_VERSION=0.0.1
```

Comment:
Now this is pinned. That means we do not need to check for updates if the file is already present in the volume. I am not too concerned about not using latest as we made only one commit in the last year and I do not expect to see anything more frequent.

@BYK, @armenzg  what do you think? I can switch to latest if needed, but that would mean checking for updates every time.

Reply:
because it is substantially different in a way that I don't think the owners are planning to integrate (we added quite a few features that make sense for us for optimization).
We never really had time to try to push those back to upstream but I would like to do that in 2021 or to start over with our implementation. 
In the end this library is 800 lines of C, so it is quite simple to maintain, thus I was not too concerned so far.

Reply:
I actually prefer using latest as that's what we do for development for all other things. We use `:nightly` tags. https://github.com/getsentry/wal2json/releases/latest would redirect you to the latest release at all times. We can then version check.

Reply:
Ok, I made some changes, this is a way in between.
It tries to find the latest version.
If it can it checks if we have that version locally. 
If not it tries to download 
If yes it uses that and does not try to download another version
If it cannot get from github the latest version id, it checks if we have a valid version locally
If there is one it uses that one 
if not the last option is to look for a wal2json still installed for some reason
At this point it is the end and it gives up.